tree/tree.go

```diff
@@ -1,3 +1,27 @@
+// Package tree provides an API to create printable tree-like structures that
+// can be included in any command line application. It goes something like:
+//
+//	t := tree.New().
+//		Root(".").
+//		Item("Item 1").
+//		Item(
+//		tree.New().Root("Item 2").
+//			Item("Item 2.1").
+//			Item("Item 2.2").
+//			Item("Item 2.3"),
+//		).
+//		Item(
+//		tree.New().
+//			Root("Item 3").
+//			Item("Item 3.1").
+//			Item("Item 3.2"),
+//		)
+//
+//	fmt.Println(t)
+//
+// If you're looking to create a list, you can use the list package which wraps
```

Comment:
could probably add more info here on the difference between the two, but I *think* this one sentence + the examples showing some differences in implementation might be enough?